### PR TITLE
add post on performance now conference 2019

### PR DESCRIPTION
**Pull request by paulwhitehead01**
first draft

### CODE REVIEW

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+As a performance engineer with a focus on ensuring our websites are stable and can cope with predicted load, it’s easy to lose sight of the fact that front-end performance is of equal importance and shouldn’t be seen as secondary concern or problem for only front-end developers to address. After all, it’s no good having a 100% uptime if customers are turning away because the site is slow and unresponsive.
+
+![Disciplined delivery](/images/performance-now-conf-04.jpg)
+
+> “The quickest request is the one never made” - Henri Helvetica
+
+The opening talk of the conference was from Henri Helvetica, a freelance developer, entitled [“A decade of disciplined deliver”](https://www.youtube.com/watch?v=HC1eVj5cQOo) in which he walked through the 14 rules set out in Sounders seminal publication to see how relevant they are today.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: “delivery”

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+> “We’ve built a web that largely dismisses affordable typical smartphones and the people that use them” - Tim Kadlec
+
+Tim Kadlec, a performance consult and trainer, gave a talk entitled [“When JavaScript bytes”](https://www.youtube.com/watch?v=JvJ0v5OohNg) in which he highlighted the cost of JavaScript and the practical ways it can be reduced. Tim made an interesting comparison between the 1.7mb of code that helped land man on the moon and the 1.8mb shipped for your average mobile site. A big difference in code and purpose but it does show what can be done where necessity dictates. Tim emphasises the fact that JavaScript byte-for-byte is the most expensive resource on the web with x3 performance penalty in terms of network, on-device and execution cost. With many sites neglecting to follow best practices like compression and reduction in bundles sizes, coupled with widespread use of lower spec mobile devices on poor networks, there should be no room for complacency. Tim suggests enforcing strict limits on code size from outset else continuing to chip away at legacy code while there is the opportunity to do so.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: capitalise MB

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+> “Everyone who works on a web product shares ownership of performance and security - whether they know it or not" - Simon Hearne
+
+Simon Hearn, Web Performance Solutions Engineer as Akamai, gave us a talk entitled ["Deep dive into third-party performance"](https://www.youtube.com/watch?v=uXv9JFvrnwo). Through looking at post-mortems of recent incidents Simon sort to equip his audience with the stories, tools, and techniques to manage third-party content and what to look out for when evaluating a new third-party service. As you’d expect, third-party web content has seen a similar trajectory of growth to that of JavaScript in recent years with the median website now made up of 37% third-party requests. In hid talk, Simon acknowledges that third-party content, though a source of revenue, can become a source of irritation that creates friction between development and marketing teams. He encourages everyone to become third-party subject matter experts to advance both their careers and the products they work on.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: s/sort/sought/
s/hid/his/

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Over the years I’ve made use of many tools to gain a better understanding of site performance - here are a few of my favourites that are free, easy to use, and provide a wealth of advice and insights :
+
+* [Sitespeed.io](https://www.sitespeed.io): Not mentioned at the conference but one of my favourites. Sitespeed.io is a set of Open Source tools that makes it easy to monitor and measure the performance of your web site. Sitespeed.io is the complete toolbox to test and  monitor your performance or checkout how your competition is doing.
+
+* [Webpagetest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: Consistent capitalisation - WebPageTest

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+![Performance Budgets](/images/performance-now-conf-03.png)
+ 
+> “Everyone who touches a page should care about the performance of that page” - Tammy Everts
+
+Tammy Everts talk entitled [“The 7 habits of highly effective performance teams”](https://www.youtube.com/watch?v=SE0HhF4TO0Q) shared tips and best practice gleaned from conversations with companies leading the way in web performance. These companies had one thing in common - they had a strong culture of performance. 
+
+The 7 habits covered in the talk were as follows:
+
+* 1 - Performance is a marathon, not a sprint
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: If this is meant to be an ordered list, use [markdown format](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+---
+layout:     post
+title:      Performance.Now() Conference 2019.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]: Consistent formatting throughout the article - should be lower case even in titles, and with the parentheses, so `performance.now()`

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 1 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: done

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 2 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: fixed

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 3 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: fixed

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 4 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: fixed

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 5 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: fixed

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 6 above)
Comment (paulwhitehead01) [reply _posts/2019-12-20-performance-now-conference-2019.md]: fixed

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+---
+layout:     post
+title:      performance.now() Conference 2019.
+date:       2019-12-20
+summary:    Key takeaways from the 2019 Performance.Now conference in Amsterdam
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]:
```suggestion
summary:    Key takeaways from the 2019 performance.now() conference in Amsterdam
```

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+image:      performance-now-conf-01.jpg
+category:   Conferences
+tags:       performance, devops, web development
+author:     paul_whitehead
+---
+
+### Introduction
+I recently attended the [Performance.Now() conference](https://perfnow.nl/) in Amsterdam courtesy of Sky Betting & Gaming’s [Tech Ninja Fund](https://www.skybetcareers.com/your-learning). Performance.now() was a single-track conference with fourteen world-class speakers, covering today’s most important web performance insights. As a performance test engineer with keen interesting in front-end performance, I was looking forward to catching up on the latest development, ideas, and approaches in the industry and also hoped to pick up some tips along the way.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]:
```suggestion
I recently attended the [performance.now() conference](https://perfnow.nl/) in Amsterdam courtesy of Sky Betting & Gaming’s [Tech Ninja Fund](https://www.skybetcareers.com/your-learning). performance.now() was a single-track conference with fourteen world-class speakers, covering today’s most important web performance insights. As a performance test engineer with keen interesting in front-end performance, I was looking forward to catching up on the latest development, ideas, and approaches in the industry and also hoped to pick up some tips along the way.
```

File: _posts/2019-12-20-performance-now-conference-2019.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+Over the years I’ve made use of many tools to gain a better understanding of site performance - here are a few of my favourites that are free, easy to use, and provide a wealth of advice and insights :
+
+* [Sitespeed.io](https://www.sitespeed.io): Not mentioned at the conference but one of my favourites. Sitespeed.io is a set of Open Source tools that makes it easy to monitor and measure the performance of your web site. Sitespeed.io is the complete toolbox to test and  monitor your performance or checkout how your competition is doing.
+
+* [WebPageTest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```
Comment (cmbuckley) [inline _posts/2019-12-20-performance-now-conference-2019.md]:
```suggestion
* [WebPageTest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPageTest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```

File: _posts/2019-12-20-performance-now-conference-2019.md (hunk 4 above)
Comment (ultrafez) [reply _posts/2019-12-20-performance-now-conference-2019.md]: As confusing as the naming is, the service name is capitalised WebPagetest - see [their about page](https://www.webpagetest.org/about).

```suggestion
* [WebPagetest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```

**Pull request merged**